study-builder/studies/pancan/snippets/common/cancer-picklist-groups-sarcomas.conf

```diff
@@ -40,376 +42,388 @@
     },
     {
-      "stableId": "C_SARCOMAS_MFH_MALIGNANT_FIBROUS",
+      "stableId": "C_SARCOMAS_GIST_GASTROINTEST_STROMAL",
       "optionLabelTemplate": {
         "templateType": "TEXT",
-        "templateText": "$c_sarcomas_mfh_malignant_fibrous",
+        "templateText": "$c_sarcomas_gist_gastrointest_stromal",
         "variables": [
           {
-            "name": "c_sarcomas_mfh_malignant_fibrous",
+            "name": "c_sarcomas_gist_gastrointest_stromal",
             "translations": [
-              { "language": "en", "text": ${i18n.en.cancer.c_sarcomas_mfh_malignant_fibrous} },
-              { "language": "es", "text": ${i18n.es.cancer.c_sarcomas_mfh_malignant_fibrous} },
+              { "language": "en",
+                "text": ${i18n.en.cancer.c_sarcomas_gist_gastrointest_stromal} },
+              { "language": "es",
+                "text": ${i18n.es.cancer.c_sarcomas_gist_gastrointest_stromal} },
             ]
           }
         ]
       }
     },
     {
-      "stableId": "C_SARCOMAS_MESENCHYMAL_CHONDROSARCOMA",
+      "stableId": "C_SARCOMAS_HPC_HEMANGIOPERICYTOMA",
       "optionLabelTemplate": {
         "templateType": "TEXT",
-        "templateText": "$c_sarcomas_mesenchymal_chondrosarcoma",
+        "templateText": "$c_sarcomas_hpc_hemangiopericytoma",
         "variables": [
           {
-            "name": "c_sarcomas_mesenchymal_chondrosarcoma",
+            "name": "c_sarcomas_hpc_hemangiopericytoma",
             "translations": [
-              { "language": "en", "text": ${i18n.en.cancer.c_sarcomas_mesenchymal_chondrosarcoma} },
-              { "language": "es", "text": ${i18n.es.cancer.c_sarcomas_mesenchymal_chondrosarcoma} },
+              { "language": "en",
+                "text": ${i18n.en.cancer.c_sarcomas_hpc_hemangiopericytoma} },
+              { "language": "es",
+                "text": ${i18n.es.cancer.c_sarcomas_hpc_hemangiopericytoma} },
             ]
           }
         ]
       }
     },
     {
-      "stableId": "C_SARCOMAS_MYXOSARCOMA",
+      "stableId": "C_SARCOMAS_KS_KAPOSI_SARCOMA",
       "optionLabelTemplate": {
         "templateType": "TEXT",
-        "templateText": "$c_sarcomas_myxosarcoma",
+        "templateText": "$c_sarcomas_ks_kaposi_sarcoma",
         "variables": [
           {
-            "name": "c_sarcomas_myxosarcoma",
+            "name": "c_sarcomas_ks_kaposi_sarcoma",
             "translations": [
-              { "language": "en", "text": ${i18n.en.cancer.c_sarcomas_myxosarcoma} },
-              { "language": "es", "text": ${i18n.es.cancer.c_sarcomas_myxosarcoma} },
+              { "language": "en",
+                "text": ${i18n.en.cancer.c_sarcomas_ks_kaposi_sarcoma} },
+              { "language": "es",
+                "text": ${i18n.es.cancer.c_sarcomas_ks_kaposi_sarcoma} },
             ]
           }
         ]
       }
     },
     {
-      "stableId": "C_SARCOMAS_S_OSTEO_SARCOMA",
+      "stableId": "C_SARCOMAS_S_LEIOMYO_LMS_SARCOMA",
```

Comment:
I didn't rename this stable id since it was modified in `develop` after I branched. Should be easy enough to go back and update after the fact, but it can be left as-is.